src/infrastructure/github.ts

```diff
@@ -23,6 +26,17 @@ export class GitHub {
       pull_number,
     })
   }
+
+  async getPullRequestsAssociatedWithCommit(): Promise<PullsGetAssociatedWithCommitResponse> {
```

Comment:
I would return here the data object of "pullsAssociatedWithCommit.data", and call it `AssociatedPullRequests' or something, to avoid passing the response object outside of the GitHub API infra. 

This is a design error we have in Standard Adoption Tool (passing the whole response object outside of the infra module). An Advantage of Typescript is that we can see more clearly when we type things.